clarity_ext/pdf.py

```diff
@@ -2,8 +2,8 @@
 
 
```

Comment:
Regarding this, I feel that it's most important that developers can quickly find their way through the project - without running a search or viewing the project in a compiled class representation in some IDE.

Even if it doesn't contain much at the moment, the module encapsulates entirely what it's there for. So if someone needs to solve a bug regarding pdfs, he can assume that he only has to read through this small module.

The problem I have with packages as directories is that it's not very clear what should go where. I think a module that's put in a directory where it doesn't really belong is better off in the root. PDF parsing is not more miscellaneous than e.g. unit_conversion, so putting them into e.g. something called `misc` begs the question where to draw that line.

I think it might make sense to move this into a package called e.g. `io`. But currently there is no other file that should clearly go into that package, so I suggest we wait with that.